### PR TITLE
fix clouds-maas.md; issue #1126

### LINKED ISSUE #1126: clouds-maas page needs work, add-cloud first
I noticed, with b7, that 'maas' is not a built-in cloud anymore (does not show up in `list-clouds`). Now the [clouds-maas](https://jujucharms.com/docs/devel/clouds-maas) page is broken: need to `add-cloud` before `add-credential`.

There are also other significant and minor mistakes that need to be corrected. The text requires some clean up too.

### PR DESCRIPTION
**Pull request by pmatulis**
Fixes #1126 

Review @juju/docs

### CODE REVIEW

File: src/en/clouds-maas.md [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
-definition of a MAAS to Juju's known clouds.
-
-To do this you should create a YAML format file of the type shown below. For 
-MAAS clouds the only thing that varies is the endpoint and the name, since they
-use the same authentication method. 
-For example, we could create the file maas-clouds.yaml:
-  
+For each MAAS you may have, you will need to define it so it may become a
+member of Juju's known clouds. This is done with the help of a YAML file. All
+what varies is the endpoint and the name, since they use the same
```
Comment (matthewhelmke) [inline src/en/clouds-maas.md]: :s/what/that

**Comment by matthewhelmke** [comment]
Only one comment. After you fix the typo, you are good to go.
:cake:

**Comment by pmatulis** [comment]
Thank you. Merging now.

**Comment by evilnick** [comment]
Thanks. I hope this is settled on now!

BTW, I don't think you *have* to define maas clouds, you can still supply the endpoint at bootstrap, but that is a minor point for another day.

**Comment by pmatulis** [comment]
It will need to change again when the interactive nature of `juju bootstrap` comes online.

**Pull request merged**